2022-04-08 10:44:08,934 INFO  ==============================================================
2022-04-08 10:44:08,935 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
2022-04-08 10:44:08,937 INFO  Dir:   software_management_end_to_end/
2022-04-08 10:44:08,937 INFO  Title: Install package to directory specified in tedge config set tmp.path
2022-04-08 10:44:08,938 DEBUG Execution order hint: 0.0
2022-04-08 10:44:08,939 INFO  ==============================================================
2022-04-08 10:44:10,413 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
2022-04-08 10:44:10,414 DEBUG EnvironmentC8y Setup
2022-04-08 10:44:10,415 INFO  EnvironmentC8y Setup
2022-04-08 10:44:10,416 DEBUG Process parameters for systemctl<serv_mapper1>
  command line : /usr/bin/systemctl status tedge-mapper-c8y
    arg #1     : status
    arg #2     : tedge-mapper-c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/serv_mapper1.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/serv_mapper1.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-04-08 10:44:10,427 DEBUG Waiting up to 600 secs for process systemctl<serv_mapper1> (pid 32300)
2022-04-08 10:44:11,037 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-04-08 10:44:11,167 DEBUG Process parameters for sudo<tedge_connect_c8y>
  command line : /usr/bin/sudo /usr/bin/tedge connect c8y
    arg #1     : /usr/bin/tedge
    arg #2     : connect
    arg #3     : c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/tedge_connect_c8y.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/tedge_connect_c8y.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-04-08 10:44:11,179 DEBUG Waiting up to 600 secs for process sudo<tedge_connect_c8y> (pid 32304)
2022-04-08 10:44:25,179 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 14 secs
2022-04-08 10:44:25,183 DEBUG Process parameters for sudo<tedge_connect_c8y_test>
  command line : /usr/bin/sudo /usr/bin/tedge connect c8y --test
    arg #1     : /usr/bin/tedge
    arg #2     : connect
    arg #3     : c8y
    arg #4     : --test
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/tedge_connect_c8y_test.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/tedge_connect_c8y_test.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-04-08 10:44:25,191 DEBUG Waiting up to 600 secs for process sudo<tedge_connect_c8y_test> (pid 32414)
2022-04-08 10:44:25,799 INFO  Executed sudo<tedge_connect_c8y_test>, exit status 0
2022-04-08 10:44:25,802 DEBUG Process parameters for systemctl<serv_mosq2>
  command line : /usr/bin/systemctl status mosquitto
    arg #1     : status
    arg #2     : mosquitto
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/serv_mosq2.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/serv_mosq2.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-04-08 10:44:25,809 DEBUG Waiting up to 600 secs for process systemctl<serv_mosq2> (pid 32419)
2022-04-08 10:44:26,316 INFO  Executed systemctl<serv_mosq2>, exit status 0
2022-04-08 10:44:26,318 DEBUG Process parameters for systemctl<serv_mapper3>
  command line : /usr/bin/systemctl status tedge-mapper-c8y
    arg #1     : status
    arg #2     : tedge-mapper-c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/serv_mapper3.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/serv_mapper3.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-04-08 10:44:26,326 DEBUG Waiting up to 600 secs for process systemctl<serv_mapper3> (pid 32420)
2022-04-08 10:44:26,783 INFO  Executed systemctl<serv_mapper3>, exit status 0
2022-04-08 10:44:28,815 DEBUG Starting new HTTPS connection (1): thin-edge-io.eu-latest.cumulocity.com:443
2022-04-08 10:44:29,011 DEBUG https://thin-edge-io.eu-latest.cumulocity.com:443 "GET /devicecontrol/operations?deviceId=17063047&pageSize=1&revert=true&dateFrom=1970-01-01T00%3A00%3A00.000Z HTTP/1.1" 200 395
2022-04-08 10:44:29,025 DEBUG Final URL of the request: https://thin-edge-io.eu-latest.cumulocity.com/devicecontrol/operations?deviceId=17063047&pageSize=1&revert=true&dateFrom=1970-01-01T00%3A00%3A00.000Z
2022-04-08 10:44:29,029 ERROR No operations found, assuming it passed
2022-04-08 10:44:29,046 DEBUG Starting new HTTPS connection (1): thin-edge-io.eu-latest.cumulocity.com:443
2022-04-08 10:44:29,231 DEBUG https://thin-edge-io.eu-latest.cumulocity.com:443 "GET /inventory/managedObjects/17063047 HTTP/1.1" 200 1336
2022-04-08 10:44:29,247 WARN  TypeError occurred while running test - 'NoneType' object is not iterable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 49, in setup
    self.assertThat("False == value", value=self.check_is_installed("rolldice"))
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 398, in check_is_installed
    for package in package_list:
TypeError: 'NoneType' object is not iterable
2022-04-08 10:44:29,266 WARN  TypeError: 'NoneType' object is not iterable ... blocked 
2022-04-08 10:44:29,267 DEBUG --- test cleanup
2022-04-08 10:44:29,268 INFO  
2022-04-08 10:44:29,268 INFO  cleanup:
2022-04-08 10:44:29,269 DEBUG Running registered cleanup function: <bound method AptInstallWithDownloadPath.mysmcleanup of <AptInstallWithDownloadPath object at 0x75d7fcb0>>
2022-04-08 10:44:29,271 DEBUG Process parameters for sudo
  command line : /usr/bin/sudo rm -rf /tedge_download_path_test
    arg #1     : rm
    arg #2     : -rf
    arg #3     : /tedge_download_path_test
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux
  stdout       : None
  stderr       : None
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-04-08 10:44:29,279 DEBUG Waiting up to 600 secs for process sudo (pid 32421)
2022-04-08 10:44:29,788 INFO  Executed sudo, exit status 0
2022-04-08 10:44:29,790 DEBUG Process parameters for sudo
  command line : /usr/bin/sudo /usr/bin/tedge config set tmp.path None
    arg #1     : /usr/bin/tedge
    arg #2     : config
    arg #3     : set
    arg #4     : tmp.path
    arg #5     : None
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux
  stdout       : None
  stderr       : None
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-04-08 10:44:29,798 DEBUG Waiting up to 600 secs for process sudo (pid 32423)
2022-04-08 10:44:30,308 INFO  Executed sudo, exit status 0
2022-04-08 10:44:30,422 DEBUG Process parameters for sudo<tedge_disconnect_c8y>
  command line : /usr/bin/sudo /usr/bin/tedge disconnect c8y
    arg #1     : /usr/bin/tedge
    arg #2     : disconnect
    arg #3     : c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/tedge_disconnect_c8y.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/tedge_disconnect_c8y.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-04-08 10:44:30,430 DEBUG Waiting up to 600 secs for process sudo<tedge_disconnect_c8y> (pid 32428)
2022-04-08 10:44:34,500 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-04-08 10:44:34,607 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
2022-04-08 10:44:34,609 INFO  Delaying execution by 8 seconds
2022-04-08 10:44:42,620 DEBUG Process parameters for sudo<tedge_connect_c8y>
  command line : /usr/bin/sudo /usr/bin/tedge connect c8y
    arg #1     : /usr/bin/tedge
    arg #2     : connect
    arg #3     : c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/tedge_connect_c8y.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/tedge_connect_c8y.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-04-08 10:44:42,629 DEBUG Waiting up to 600 secs for process sudo<tedge_connect_c8y> (pid 32471)
2022-04-08 10:44:56,619 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 14 secs
2022-04-08 10:44:56,622 DEBUG Running registered cleanup function: <bound method EnvironmentC8y.myenvcleanup of <AptInstallWithDownloadPath object at 0x75d7fcb0>>
2022-04-08 10:44:56,623 DEBUG EnvironmentC8y Cleanup
2022-04-08 10:44:56,754 DEBUG Process parameters for sudo<tedge_disconnect_c8y>
  command line : /usr/bin/sudo /usr/bin/tedge disconnect c8y
    arg #1     : /usr/bin/tedge
    arg #2     : disconnect
    arg #3     : c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/tedge_disconnect_c8y.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/tedge_disconnect_c8y.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-04-08 10:44:56,764 DEBUG Waiting up to 600 secs for process sudo<tedge_disconnect_c8y> (pid 32587)
2022-04-08 10:45:00,762 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-04-08 10:45:00,765 DEBUG Process parameters for systemctl<serv_mapper5>
  command line : /usr/bin/systemctl status tedge-mapper-c8y
    arg #1     : status
    arg #2     : tedge-mapper-c8y
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/serv_mapper5.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/Output/linux/serv_mapper5.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
2022-04-08 10:45:00,773 DEBUG Waiting up to 600 secs for process systemctl<serv_mapper5> (pid 32628)
2022-04-08 10:45:01,380 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-04-08 10:45:01,382 DEBUG ProcessUser cleanup function done.
2022-04-08 10:45:01,385 INFO  
2022-04-08 10:45:01,386 INFO  Test duration: 52.47 secs
2022-04-08 10:45:01,387 INFO  Test final outcome:  BLOCKED
2022-04-08 10:45:01,387 INFO  Test outcome reason: TypeError: 'NoneType' object is not iterable
2022-04-08 10:45:01,388 INFO  
